worlds/messenger/docs/en_The Messenger.md

```diff
@@ -59,12 +59,14 @@ for it. The groups you can use for The Messenger are:
 * After reaching ninja village a teleport option is added to the menu to reach it quickly
 * Toggle Windmill Shuriken button is added to option menu once the item is received
 * The mod option menu will also have a hint item button, as well as a release and collect button that are all placed when
-  the player fulfills the necessary conditions.
+the player fulfills the necessary conditions.
```

Comment:
As per style guide the old indentation was correct. There are a lot more you "fixed" in this and setup_en.md